cloud/src/meta-store/keys.h

```diff
@@ -218,6 +218,9 @@ using MetaPendingDeleteBitmapInfo = BasicKeyInfo<24 , std::tuple<std::string, in
 //                                                      0:instance_id 1:db_id  2:job_id
 using RLJobProgressKeyInfo = BasicKeyInfo<25, std::tuple<std::string, int64_t, int64_t>>;
 
+//                                                      0:instance_id 1:db_id  2:job_id
+using StreamingJobMetaKeyInfo = BasicKeyInfo<52, std::tuple<std::string, int64_t, int64_t>>;
```

Comment:
what is the difference between  StreamingJobMetaKeyInfo and RLJobProgressKeyInfo?
if they all components are identical we should use RLJobProgressKeyInfo instead.